tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+import logging
+from typing import Dict, List, Optional, Tuple
+
+import git
+import requests
+import yaml
+
+AIRBYTE_REPO = git.Repo(".")
+OSS_CATALOG_URL = "https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/oss_catalog.json"
+CONNECTOR_PATH_PREFIX = "airbyte-integrations/connectors"
+SOURCE_CONNECTOR_PATH_PREFIX = CONNECTOR_PATH_PREFIX + "/source-"
+ACCEPTANCE_TEST_CONFIG_FILE_NAME = "acceptance-test-config.yml"
+AIRBYTE_DOCKER_REPO = "airbyte"
+
+
+def download_catalog(catalog_url):
+    response = requests.get(catalog_url)
+    return response.json()
+
+
+OSS_CATALOG = download_catalog(OSS_CATALOG_URL)
```

Comment:
Nit: Since you are using the published catalog, there's a race-condition if /this/ PR is changing the release stage that the test won't run. Could we look at the local YML files in this branch instead?

Reply:
Oh yes, you're right. Moreover, the approach I took to compute the diff would also detect connector changes made on master after the PR was opened and not rebased... I improved the behavior in https://github.com/airbytehq/airbyte/pull/19383/commits/929e3b0fbac15c2bf278ca553c1536c0a618117b